styles/style.css

```diff
@@ -18,25 +21,63 @@
   margin: 0 auto;
 }
 
-.section-icon {
+.container {
+  display: flex;
+  justify-content: center;
   max-width: 1440px;
   box-sizing: border-box;
-  margin: 0 auto;
   padding: 80px 88px 0 104px;
 }
 
 .icon {
-  display: inline-block;
   width: 296px;
   height: 176px;
   text-align: center;
-  float: left;
   margin-right: 16px;
   margin-bottom: 122px;
 }
 
 .caption {
   font-family: 'Roboto', sans-serif;
   font-weight: 400;
+  font-size: 20px;
   margin-top: 48px;
+  line-height: 28px;
+}
+
+/* footer */
+.footer {
+  background-color: #10367C;
+  color: #ffffff;
+}
+
+.container-footer {
+  display: flex;
+  justify-content: space-between;
+  padding: 120px 272px 64px 104px;
+}
+
+.footer-list {
+  list-style: none;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 24px;
```

Comment:
This is **a bad practice**, to use absolute values for line-height
For example if you need to change font-size for mobiles - `line-height: 24px` can look badly
Using of relative values will be better
```
line-height: 2.5;
line-height: 3em;
line-height: 150%;
```